Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN apt-get update \
     /var/tmp/*
 
 COPY . /app
+RUN pip install --upgrade pip==23.2.1
```

Comment:
From all the Dockerfiles I've ever seen, they just upgrade and not specify a specific version.
I agree that it's the practice with libraries, but in `pip` I think latest is preferable.
Anyhow, approving, up to you.